README.md

```diff
@@ -62,70 +100,66 @@ using       ton = unlib::kilo<kilogram>;       // humans don't use megagram
 A quantity combines a [unit](#units) (_mass_), a [scale](#scales) (_kilo_), a specific [value type](#values) (_double_), and an optional [tag](#tags) (_reactive_): 
 
 ```c++
-using kilogram = unlib::quantity<unlib::mass,unlib::kilo,double>. 
+using kilogram = unlib::quantity<unlib::mass,unlib::kilo,double>;
```

Comment:
this is wrong, the base unit of mass is already kilogram, what you define here is megagram. Also, `unlib::kilo` is an alias for `scale_by`, what you want here is the scaling explicitly.
```suggestion
using kilogram = unlib::quantity<unlib::mass,unlib::no_scaling,double>;
```

Reply:
Ugh. OK, as we talked about, `kilogram<>` will again become `kilo<gram<>>`, which would make `unlib::kilo_scaling` the correct identifier here.

However, since this will be done on a different branch, I just avoided the issue by using kilometer as an example here:

```suggestion
using kilometer = unlib::quantity<unlib::length,unlib::kilo_scaling,double>;
```